io.github.hakuneko.HakuNeko.json

```diff
@@ -0,0 +1,54 @@
+{
+	"app-id": "io.github.hakuneko.HakuNeko",
+	"base": "io.atom.electron.BaseApp",
```

Comment:
Any reason for not using  org.electronjs.Electron2.BaseApp ? unless the app depends on older electron version that still requires gtk2

Reply:
OK, I switched to org.electronjs.Electron2.BaseApp